cstrml/balances/src/benchmarking.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
```

Comment:
Change to our copyright

Reply:
changed. Added balances tests in our CI.